### PR TITLE
AB#28415 add pause visa card test

### PR DESCRIPTION
**Pull request by Piotrk39**
Adds Pause Visa Debit Card e2e test for both OCW and PV programme & edits the names of Payment related tests.

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I have added tests wherever relevant
- [x] I have made sure that all automated checks pass before requesting a review
- [x] I do not need any deviation from our PR guidelines

### CODE REVIEW

File: e2e/pages/RegistrationDetails/RegistrationDetailsPage.ts [hunk 1]
```diff
@@ -258,6 +258,28 @@ class RegistrationDetails {
     }
   }
```
Comment (RubenGeo) [inline e2e/pages/RegistrationDetails/RegistrationDetailsPage.ts:260]: Your RegistrationDetailsPage.ts file is becoming quite long. Maybe it makes sense to consider creating a seperate RegistrationPhysicalCardOverview.ts file and move all card functionality to there.  

I am not 100% sure about the structure you should try to adhere to in playwright e2e code. But for example in our fronted the RegistrationDetailsPage is seperated out in a lot of components .

File: e2e/tests/121-Portal/MakeNewPayment/OCW/PauseCardOCW.spec.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  // Login
+  const loginPage = new LoginPage(page);
+  await page.goto('/login');
+  await loginPage.login(
+    process.env.USERCONFIG_121_SERVICE_EMAIL_ADMIN,
+    process.env.USERCONFIG_121_SERVICE_PASSWORD_ADMIN,
+  );
+});
+
+test('[28441] Pause Visa debit cards', async ({ page }) => {
```
Comment (RubenGeo) [inline e2e/tests/121-Portal/MakeNewPayment/OCW/PauseCardOCW.spec.ts:31]: Does it make sense to also add an unpause test? I think you can largely reuse what you already have.

**Review by RubenGeo — CHANGES_REQUESTED** [review]
Functionally works great! I just have some small comments :)

File: e2e/pages/RegistrationDetails/RegistrationDetailsPage.ts (hunk 1 above)
Comment (Piotrk39) [reply e2e/pages/RegistrationDetails/RegistrationDetailsPage.ts:260]: That is actually a good idea to do that. Thanks

File: e2e/tests/121-Portal/MakeNewPayment/OCW/PauseCardOCW.spec.ts (hunk 2 above)
Comment (Piotrk39) [reply e2e/tests/121-Portal/MakeNewPayment/OCW/PauseCardOCW.spec.ts:31]: It does indeed but I would like to add it in another PR.

File: e2e/pages/RegistrationDetails/RegistrationDetailsPage.ts [hunk 3]
```diff
@@ -258,6 +258,28 @@ class RegistrationDetails {
...
 
+  async pauseVisaDebitCard() {
+    try {
+      const activeCard = this.debitCardStatus.filter({ hasText: 'Active' });
+      await activeCard.waitFor({ state: 'visible' });
+      await activeCard.click();
+
+      const issueNewCardButton = this.page.getByRole('button', {
```
[the remark this answers is not in the document]
Comment (Piotrk39) [reply e2e/pages/RegistrationDetails/RegistrationDetailsPage.ts:267]: Done

File: e2e/tests/121-Portal/MakeNewPayment/OCW/PauseCardOCW.spec.ts (hunk 2 above)
Comment (Piotrk39) [reply e2e/tests/121-Portal/MakeNewPayment/OCW/PauseCardOCW.spec.ts:31]: I added it as an extra step to validate also the unpause

File: e2e/pages/RegistrationDetails/RegistrationDetailsPage.ts (hunk 1 above)
Comment (Piotrk39) [reply e2e/pages/RegistrationDetails/RegistrationDetailsPage.ts:260]: I have added new file and according to your input I'll add more files depending on functionality.

**Review by RubenGeo — APPROVED** [review]

**Pull request merged**